### PR TITLE
Added support for nvd mirror and central nvd server

### PR DESCRIPTION
**Pull request by oconnormi**
#### What does this PR do?

* Disables unnecessary owasp scanners
* Adds properties for owasp nvd mirror
* Adds properties for centralized nvd server

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
 @figliold
 @mcalcote

#### How should this be tested? (List steps with links to updated documentation)

Run owasp scan in CI

#### Tasks
- [x] Ensure pipeline runs owasp
- [x] Add owasp aggregation to pipeline
- [x] Add post step to archive the owasp report

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -249,6 +249,54 @@
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
                     <version>${dependency.check.maven.plugin.version}</version>
+                    <configuration>
```
Comment (figliold) [inline pom.xml]: Do we need failBuildOnCVSS?

**Comment by oconnormi** [comment]
Build Now

File: pom.xml (hunk 1 above)
Comment (oconnormi) [reply pom.xml]: we do i was trying to figure  out the CVSS threshold

**Comment by oconnormi** [comment]
Build Now

**Pull request merged**